.github/actions/issue-in-project/src/main.ts

```diff
@@ -0,0 +1,196 @@
+import { getInput, info, setOutput } from '@actions/core'
+import { getOctokit } from '@actions/github'
+import { LIB_VERSION } from '@src/version'
+import { Command } from 'commander'
+import { readFileSync } from 'fs'
+import { homedir } from 'os'
+import { join } from 'path'
+
+// read in token
+let token: string
+try {
+  token = getInput('github_token', { required: true })
+} catch (e1) {
+  try {
+    token = readFileSync(join(homedir(), '.github_token'), 'utf8').trim()
+  } catch (e2) {
+    if (e2 instanceof Error && e2.name == 'ENOENT') {
+      // ENOENT: file non-existent
+      throw e1
+    } else {
+      throw e2
+    }
+  }
+}
+
+// initialize ocotokit
+const octokit = getOctokit(token)
+
+interface Options {
+  org: string | null
+  user: string | null
+  repo: string | null
+}
+interface Issues {
+  [databaseId: number]: {
+    id: number
+    title: string
+  }
+}
+
+async function projectIssues(
+  project: number,
+  options: Options,
+): Promise<Issues> {
+  // common part of query
+  interface Content {
+    content: {
+      databaseId: number
+      number: number
+      title: string
+    }
+  }
+  interface Cards {
+    cards: { nodes: Array<Content> }
+  }
+  interface Project {
+    project: { columns: { nodes: Array<Cards> } }
+  }
+  const query: string = `
+    project(number: ${project}) {
+      columns(first: 10) {
```

Comment:
What happens if there are more than 10 columns?

Reply:
GH's GraphQL requires we provide a pagination limit for the columns parameter:
```
"You must provide a `first` or `last` value to properly paginate the `columns` connection."
```

I just randomly picked 10 as currently we don't have more than 10 columns on any board, I can also up this to the largest value allowed (100, see https://docs.github.com/en/graphql/overview/resource-limitations#node-limit).

Reply:
Let's do 100 then, no need to add subtle issues like that early on, 10 isn't completely unrealistic.

Reply:
Ended up reworking it to use pagination so we're safe no matter how many columns we have!! 😄